beetsplug/discogs.py

```diff
@@ -71,6 +73,7 @@ def setup(self, session=None):
         # Try using a configured user token (bypassing OAuth login).
         user_token = self.config['user_token'].as_str()
         if user_token:
+            self.rate_limit_per_minute = 60
```

Comment:
Perhaps a comment here would help explain why the rate limit is different when we have a user token.